src/test/java/org/wiremock/extension/jwt/JwtHelperAcceptanceTest.java

```diff
@@ -208,8 +228,7 @@ void supports_RS256_signed_tokens() {
                             secret))))
             .build());
 
-    DecodedJWT decodedJwt =
-        Assertions.assertDoesNotThrow(() -> verifyRs256AndDecodeForTemplate("{{jwt alg='RS256'}}"));
+    DecodedJWT decodedJwt = Assertions.assertDoesNotThrow(() -> verifyRs256AndDecodeForTemplate("{{jwt alg='RS256'}}"));
```

Comment:
There seem to be a few changes like this that are just format changes. Can you remove these for clarity?